### PR TITLE
update Vert.x MQTT server component version in docs

### PR DESCRIPTION
**Pull request by boliza**

### CODE REVIEW

File: src/main/java/io/vertx/mqtt/package-info.java [hunk 1]
```diff
@@ -27,39 +27,25 @@
  *
  * == Using Vert.x MQTT server
  *
- * As component not yet officially released in the Vert.x stack, to use the Vert.x MQTT server current snapshot version,
- * add the following repository under the _repositories_ section and the following dependency to the _dependencies_ section
+ * THis component had officially released in the Vert.x stack, just following dependency to the _dependencies_ section
```
Comment (ppatierno) [inline src/main/java/io/vertx/mqtt/package-info.java]: Just fix the typo "THis" in "this" ... thanks !

Comment (boliza) [inline src/main/java/io/vertx/mqtt/package-info.java]: OK. Thanks for you review

**Comment by cescoffier** [comment]
@ppatierno I think it can be merged, but I let have the last call.

**Comment by ppatierno** [comment]
@cescoffier sure ! I'm going to do that ;)

**Pull request merged**